### DISCUSSION TITLE
Clarification on using default local-path-provisioner

### DISCUSSION THREAD

**Discussion by PeaceRebel** [k3s-io/k3s#6490] [Q&A]
I tried creating the PVC and Pod from the k3s documentation (https://docs.k3s.io/storage#pvcyaml) which says it should work out of the box. Pod is pending and PVC provisioning failed and events has:
```
failed to provision volume with StorageClass "local-path": failed to create volume pvc-7439dfc4-76f8-4aa5-96ca-b6e6803fd41a: create process timeout after 120 seconds
 ```
 
 What is causing this? Provisioner is running in the kube-system namespace. PVC and Pod from this example are in default

**Comment by brandond** [comment]
Have you checked the logs in the local-path-provisioner pod?

**Reply by PeaceRebel** [reply]
It's not working only on nodes running coreos. I have raised an issue with local-path-provisioner. Thanks.
